roles/piwigo/defaults/main.yml

```diff
@@ -8,8 +8,8 @@ piwigo_data_directory: "{{ docker_home }}/piwigo/data"
 piwigo_photos: "{{ photos_root }}/piwigo"
 
 # uid / gid
-piwigo_user_id: "0"
-piwigo_group_id: "0"
+piwigo_user_id: "1"
```

Comment:
I don't think uid/gid 1 is the right one for this - we should probably use 1000

Reply:
Fixed